_config.yml

```diff
@@ -1,6 +1,6 @@
-title: OpenAMPProject
+title: OpenAMP Project
 description: >-
-  ArmNN and Arm Compute Library
+  OpenAMP (Open Asymmetric Multi-Processing) seeks to standardize the interactions between operating environments in a heterogeneous embedded system through open source solutions for Asymmetric MultiProcessing (AMP).
 url: "https://openampproject.org"
```

Comment:
https://www.openampproject.org ?

Reply:
[https://mlplatform.org](https://mlplatform.org) - Is this site a special case then?

Reply:
I'd need to check with @morancj to be sure but I thought that if you leave "www." off, it redirects to "www.", so it is better to have the correct FQDN there.

A bit like always having the trailing slash as well - it eliminates a round-trip between the browser and the server.

Reply:
Sure - thanks @pcolmer ! I'll update this!

Reply:
@kylekirkby as of the new static site deployment, the apex record resolves to `www.`:
```
➜ curl -IL http://openampproject.org/
HTTP/1.1 301 Moved PermanentlyContent-Length: 0
Connection: keep-alive
Date: Tue, 17 Sep 2019 14:30:12 GMT
Location: https://www.openampproject.org/
Server: AmazonS3
X-Cache: Hit from cloudfront
Via: 1.1 42322007e22ea6a235ae829b1f254f98.cloudfront.net (CloudFront)
X-Amz-Cf-Pop: MAN50-C1
X-Amz-Cf-Id: U7JoXNq3zLk3ElIFfTFaK07SgZ3DCHRTfAE1dg4WTtLZmRRB7nYXQA==
Age: 1051

HTTP/2 200 
content-type: text/html
content-length: 6750
date: Tue, 17 Sep 2019 14:30:12 GMT
cache-control: public, max-age=86400
last-modified: Tue, 17 Sep 2019 13:24:12 GMT
etag: "8153ebf309439eaebb99e44a3185e143"
server: AmazonS3
strict-transport-security: max-age=31557601; preload
x-content-type-options: nosniff
vary: Accept-Encoding
x-cache: Hit from cloudfront
via: 1.1 340acc109468df693ec024a66dbb0f1e.cloudfront.net (CloudFront)
x-amz-cf-pop: LHR61-C2
x-amz-cf-id: 8v1cTuOGjFXjcl8xDEyZXnmola4EbuCkrcl6HNNkbv5oDxStLdaQaQ==
age: 1051
```
Similar if you start with `https://openampproject.org/`. @pcolmer Agree re. trailing slash. :)